### PR TITLE
Add Json prototype for a tour

### PR DESCRIPTION
**Pull request by acousma82**
* Rename POI_tour example to tour example because a tour is not a poi
* rename root key from poi to tour
* add keys data provide and opration company
* add key "regions", representing regions where the tour passes through
* minor fixes to make the tour more comprehensive

### CODE REVIEW

File: Muster_JSON_API_Spezifikationen/main_app_server_to_mobile_app/Tour_example.json [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+            "Spreewald"
+        ],
+        "description": "Die Zeit vor der Reformation war durch eine tiefe Frömmigkeit der Menschen gekennzeichnet, die sich durch den Ausbau von Klosteranlagen durch die Mönchsorden sowie herausragende Stadtpfarrkirchen mit prächtigen Altären widerspiegelt. Im Zuge der Reformation wurden sie oftmals umgenutzt oder aufgelöst. Auf dieser Rundtour liegen sieben historische Stadtkerne mit neun Schlössern und beeindruckenden Kirchen sowie heute kulturell genutzten Klosteranlagen und spannende Museen. In Jüterbog und Mühlberg/Elbe sind die bedeutendsten Zeugnisse der Reformation zu sehen und wurden mit dem Europäischen Kulturerbesiegel dafür ausgezeichnet. Beginnend im Fläming verläuft die Radroute von Jüterbog weiter Richtung Süden zur Elbe, um dann nach einem Bogen durch die Niederlausitz wieder zum Start zurückzukehren.",
+        "thumbnail": "thumbnail_image_url",
+        "image_gallery": [
+            "image_url_1",
+            "image_url_2",
+            "image_url_3"
+        ],
+        "tags": [
```
Comment (acousma82) [inline Muster_JSON_API_Spezifikationen/main_app_server_to_mobile_app/Tour_example.json]: Anstatt nzr tags tags und type. type ist die form der tour und tag ist dann das fortbewegungsmittel und andere tags

File: Muster_JSON_API_Spezifikationen/main_app_server_to_mobile_app/Tour_example.json [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+            "Rundtour"
+        ],
+        "length": 296,
```
Comment (pwilimzig) [inline Muster_JSON_API_Spezifikationen/main_app_server_to_mobile_app/Tour_example.json]: Should be "length_km"

File: Muster_JSON_API_Spezifikationen/main_app_server_to_mobile_app/Tour_example.json [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+        "locations": { //alle  Orte der Datenbank die sich in max. Entfernung der Route befinden. die  ids aus der xml tabelle sind als id in der location tabelle nicht vorhanden. Bsp.: id=2288
```
Comment (pwilimzig) [inline Muster_JSON_API_Spezifikationen/main_app_server_to_mobile_app/Tour_example.json]: "locations" should be excluded for now

Comment (acousma82) [reply Muster_JSON_API_Spezifikationen/main_app_server_to_mobile_app/Tour_example.json]: Alle request implemented

**Pull request merged**